libraries/3d-tools/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "library-3d-tools",
+    "version": "0.0.0",
+    "private": true,
+    "scripts": {
+        "build": "vertigis-web-sdk build",
+        "start": "vertigis-web-sdk start"
+    },
+    "dependencies": {
+        "@arcgis/core": "^4.22.2",
+        "@vertigis/web": "5.16.0-rc.14825",
+        "mapillary-js": "^4.0.0"
```

Comment:
Don't think that this is supposed to be here.

Reply:
nope, removed :)